LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerEndpoints.h

```diff
@@ -30,7 +30,7 @@ struct FLootLockerServerEndPoint
     UPROPERTY(BlueprintReadWrite, EditAnywhere, BlueprintReadWrite, Category = "LootLockerServer")
     ELootLockerServerHTTPMethod requestMethod;
 
-	FString GetRequestMethodString()
+	FString GetRequestMethodString() const
```

Comment:
Intentional?

Reply:
Indeed, it was a warning that I silenced.